src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -0,0 +1,434 @@
+﻿using System;
```

Comment:
Missing License headers. Please add them on all the files

Reply:
Good catch, fixed.